<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="newButton">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>New</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/images/Images/new.png</normaloff>:/images/Images/new.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="decelerateButton">
          <property name="text">
           <string>Speed /2</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/images/Images/left-black-arrow.png</normaloff>:/images/Images/left-black-arrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseButton">
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/images/Images/pause.png</normaloff>:/images/Images/pause.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="accelerateButton">
          <property name="text">
           <string>Speed x2</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/images/Images/right-black-arrow.png</normaloff>:/images/Images/right-black-arrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="insertFourmiliereButton">
          <property name="text">
           <string>Insert Anthill</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/images/Images/fourmiliere.png</normaloff>:/images/Images/fourmiliere.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="insertBuissonButton">
          <property name="text">
           <string>Insert Bushes</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/images/Images/bushes.png</normaloff>:/images/Images/bushes.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionAccelerer"/>
    <addaction name="actionDecelerer"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="insertFourmiliere"/>
    <addaction name="insertBuisson"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInsert"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save                [Ctrl+S]</string>
   </property>
  </action>
  <action name="insertFourmiliere">
   <property name="text">
    <string>Anthill</string>
   </property>
  </action>
  <action name="insertBuisson">
   <property name="text">
    <string>Bushes</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRependre">
   <property name="text">
    <string>Reprendre               [R]</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAccelerer">
   <property name="text">
    <string>Accélérer</string>
   </property>
  </action>
  <action name="actionDecelerer">
   <property name="text">
    <string>Decelerate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
